designdocs/modular-architecture.md

```diff
@@ -87,7 +87,7 @@ flowchart TD
     SECPBC --> BC[Bouncy Castle]
     SECPFFM --> SECPC[libsecp256k1]
     CORE --> P[ProtoBuf]
-    BASE .-> S[SLF4J]
```

Comment:
At the moment, we've got these dependendies on slf4j:

```
$ grep -R slf4j base/
base/src/main/java/org/bitcoinj/base/internal/PlatformUtils.java:import org.slf4j.Logger;
base/src/main/java/org/bitcoinj/base/internal/PlatformUtils.java:import org.slf4j.LoggerFactory;
base/src/main/java/org/bitcoinj/base/package-info.java: * has no dependencies other than {@code slf4j-api} and {@code jspecify}.
base/build.gradle:    implementation 'org.slf4j:slf4j-api:2.0.16'
base/build.gradle:    testImplementation 'org.slf4j:slf4j-jdk14:2.0.16'
```

So I think it's too early to merge this commit.